great_expectations/experimental/datasources/data_asset/data_connector/google_cloud_storage_data_connector.py

```diff
@@ -66,9 +64,6 @@ def __init__(
         self._delimiter = delimiter
         self._max_results = max_results
 
-        if gcs_options is None:
-            gcs_options = {}
```

Comment:
How come we don't need this `gcs_options` anymore?

Reply:
@Kilo59 The `DataConnector` does not use them -- the `PandasGoogleCloudStorageDataSource` does.  Thanks.